lib/commons/standards/get-html-element-spec.js

```diff
@@ -0,0 +1,49 @@
+import standards from '../../standards';
+import matchesFn from '../../commons/matches';
+
+/**
+ * Return the spec for an HTML element from the html-elms standard. Since the spec is determined by the node and what attributes it has, a node is required.
+ * @param {VirtualNode} vNode The VirtualNode to get the spec for.
+ * @return {Object} The html-elms standard spec
+ */
+function getHtmlElementSpec(vNode) {
```

Comment:
I suspect we're going to need this for SVG too, lets not limit ourselves in the name

```suggestion
function getElementSpec(vNode) {
```